### ISSUE TITLE
Non-existing images are rendered with empty placeholders

### ISSUE THREAD

**Issue by thevangelist** [Lepovirta/kuvia#2]
If the image url is invalid, image doesn't display and browser renders it's default "missing img" icon.

**Comment by jpallari** [comment]
Thanks for the report!

There seems to be a way to replace the missing image with another image in JavaScript:

``` javascript
img.onerror = function() {
  img.onerror = null;
  img.src = "/path/to/alternative/image.jpg";
  return true;
};
```

I'll add an alternative image inlined into the source code, and make the image customizable with the tool.

**Comment by jpallari** [comment]
If there's a specific placeholder image you'd like to see, please post one on the comments or something. :)

**Comment by thevangelist** [comment]
I think the optimal solution would be to log the missing files to console instead of adding them to gallery

**Comment by thevangelist** [comment]
Something like: `mising image: folder/example.png` (missing images line by line)

**Comment by jpallari** [comment]
Alright. The solution would then play out as follows:
1. Gallery attempts to load the image (essentially just creates an img tag with source loaded from a static source).
2. The `onerror` handler detects the missing image.
3. The error is logged to console.
4. The missing image is removed from the gallery.
5. The next image is loaded.

This of course will be repeated on each page load because the list of files is static.

**Comment by jpallari** [comment]
This will be fixed in PR #3. Can you checkout the changes and maybe try it out?

**Comment by jpallari** [comment]
Continued in PR #3.
